reference/6/Microsoft.PowerShell.Management/Remove-Service.md

```diff
@@ -138,7 +138,8 @@ This cmdlet does not return any output.
 
 ## NOTES
 
-* To run this cmdlet on Windows Vista and later versions of the Windows operating system, start PowerShell by using the Run as administrator option.
+To run this cmdlet on Windows Vista and later versions of the Windows operating system, start
```

Comment:
Vista isn't listed as supported for PS Core. Do we need this note still?

Reply:
Fixed